content/en/news/releases/1.4.x/announcing-1.4.8/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Announcing Istio 1.4.8
+linktitle: 1.4.8
+subtitle: Patch Release
+description: Istio 1.4.8 patch release.
+publishdate: 2020-04-23
+release: 1.4.8
+aliases:
+    - /news/announcing-1.4.8
+---
+
+This release includes bug fixes to improve robustness. This release note describes what’s different
+between Istio 1.4.7 and Istio 1.4.8.
+
+The fixes below focus on various issues related to installing Istio on OpenShift with CNI. Instructions
+for installing Istio on OpenShift with CNI can be found [here](/docs/setup/additional-setup/cni/#instructions-for-istio-1-4-x-and-openshift).
```

Comment:
This will work as intended, right? I mean this will *not* link to the 1.4 version of the website, correct?

Reply:
Correct. This links to the current site. So on preliminary.istio.io it points to the link in `master` and istio.io will point to the link on `release-1.5`. Until we release 1.6 and then they'll still point to their own versions. I hope that makes sense.